packages/common/src/type-helpers.ts

```diff
@@ -12,12 +12,11 @@ export function checkExtends<_Orig, _Copy extends _Orig>(): void {
 
 export type Replace<Base, New> = Omit<Base, keyof New> & New;
 
-export type MakeOptional<Base, Keys extends keyof Base> = Omit<Base, Keys> & Partial<Pick<Base, Keys>>;
-
 export function isRecord(value: unknown): value is Record<string, unknown> {
   return typeof value === 'object' && value !== null;
 }
 
+// ts-prune-ignore-next
```

Comment:
not sure why `ts-prune` isn't able to find these relative imports like:

![image](https://user-images.githubusercontent.com/251288/193087910-e5173ee0-2a41-4a9b-929e-bac12179a3e2.png)

Reply:
I was able to remove the comment on `errorMessage` and `errorCode`.

I still get the error on `hasOwnProperty`. I get the feeling that this false-positive is related to the fact that `hasOwnProperty` is only invoked from type guards functions. But not much we can do about that.

Reply:
removed, thanks!